src/types/libdef/npm/react-testing-library_v5.x.x.js

```diff
@@ -207,6 +211,11 @@ declare module 'react-testing-library' {
       text: TextMatch,
       options?: TextMatchOptions,
     ) => ?HTMLElement,
+    getByValue: (
+      container: HTMLElement,
+      text: TextMatch,
+      options?: TextMatchOptions,
+    ) => HTMLElement,
```

Comment:
This is fine.

Normally when we change a libdef (libdef == 3rd party library definition) ourselves we move it to `src/types/libdef/npm-custom`, because all types in `src/types/libdef/npm` are downloaded, and if we leave it here it could be overriden later.

But... this is OK because 1. I move it in my other PR https://github.com/devtools-html/perf.html/pull/1645, and 2. I plan to upstream these changes to the `flow-typed` project anyway.